ansible/roles/cartodb.postgresql/tasks/main.yml

```diff
@@ -23,6 +23,9 @@
  - name: Configure PostgreSQL host-based authentication
    template: src=pg_hba.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/pg_hba.conf
 
+ - name: Configure PostgreSQL conf
+   template: src=postgresql.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/postgresql.conf
```

Comment:
Are the changes to this file small enough that the `lineinfile` ansible module would be appropriate?